contracts/ERC20Relay.sol

```diff
@@ -55,6 +55,7 @@ contract ERC20Relay is Ownable {
     }
 
     Anchor[] public anchors;
+    mapping (bytes32 => address[]) public anchorApprovals;
```

Comment:
Should approval array in Anchor above be removed as well?

Reply:
Yep. Done.